### PR TITLE
Add Japanese localization

### PR DESCRIPTION
**Pull request by shos-03**

### CODE REVIEW

File: reminders-menu-bar/Resources/ja.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+"keyboardShortcutEnableOpenShortcutOption" = "キーボード ショートカットを有効にしてReminders Menu Barを開く";
+"keyboardShortcutRestoreDefaultButton" = "デフォルトに戻す";
+"upcomingRemindersIntervalSelectionHelp" = "今後のリマインダーの表示範囲を設定";
+"upcomingRemindersTodayTitle" = "今日";
+"upcomingRemindersInAWeekTitle" = "1週間以内";
+"upcomingRemindersInAMonthTitle" = "1か月以内";
+"upcomingRemindersAllTitle" = "すべて";
+"appNoRemindersAccessAlertMessage" = "%@はリマインダーにアクセスできません";
+"appNoRemindersAccessAlertDescription" = "
+%@はリマインダーへのアクセスが必要です。%システム環境設定から、@にリマインダーへのアクセスを許可してください。";
```
Comment (DamascenoRafael) [inline reminders-menu-bar/Resources/ja.lproj/Localizable.strings:49]: The appNoRemindersAccessAlertDescription will take 2 parameters through `%@`. In your translation we have only one `%@` (looks like the second `%@` got the `%` and `@` separated in the text). 

Also, could you keep the same distribution of sentences as we have in the original text? A sentence on line 49, a sentence on line 50 and finally `";` on line 51. This helps to control whether the translations have all the necessary keys.

https://github.com/DamascenoRafael/reminders-menubar/blob/5e18be7a9456b4ad2b1392ceb211d871edc0541f/reminders-menu-bar/Resources/en.lproj/Localizable.strings#L48-L51

**Review by DamascenoRafael — COMMENTED** [review]
Thanks for your contribution!! 🎉 
I left a few suggestions for changes so we can merge it

File: reminders-menu-bar/Resources/ja.lproj/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+"showCompletedRemindersToggleButtonHelp" = "実行済みリマインダーを表示するかどうか";
+"updateAvailableNoticeButton" = "アップデートが利用可能";
+"launchAtLoginOptionButton" = "ログイン時に起動";
+"appAppearanceMenu" = "表示";
+"appAppearanceMoreOpaqueOptionButton" = "不透明";
+"appAppearanceMoreTransparentOptionButton" = "半透明";
+"showMenuBarTodayCountOptionButton" = "今日の件数をメニューバーに表示";
+"keyboardShortcutOptionButton" = "キーボードショートカット";
+"reloadRemindersDataButton" = "再読み込み";
+"appAboutButton" = "Reminders Menu Barについて";
```
[the remark this answers is not in the document]
Comment (shos-03) [reply reminders-menu-bar/Resources/ja.lproj/Localizable.strings:31]: As you can guess, I think it is necessary to make Japanese sound natural.

File: reminders-menu-bar/Resources/ja.lproj/Localizable.strings (hunk 1 above)
Comment (shos-03) [reply reminders-menu-bar/Resources/ja.lproj/Localizable.strings]: Thank you for your advice. Fixed.
#65167ae423be432fe5fabe94e5291dee0814f502

Comment (DamascenoRafael) [reply reminders-menu-bar/Resources/ja.lproj/Localizable.strings]: Awesome! Thanks

**Comment by DamascenoRafael** [comment]
Thanks again for your contribution! 🎉 
Merging :)

**Pull request merged**